samples/42.scaleout/src/main/java/com/microsoft/bot/sample/scaleout/DialogHost.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.scaleout;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.builder.TurnContextImpl;
+import com.microsoft.bot.connector.Async;
+import com.microsoft.bot.dialogs.Dialog;
+import com.microsoft.bot.dialogs.DialogState;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * The essential code for running a dialog. The execution of the dialog is treated here as a pure function call.
+ * The input being the existing (or old) state and the inbound Activity and the result being the updated (or new) state
+ * and the Activities that should be sent. The assumption is that this code can be re-run without causing any
+ * unintended or harmful side-effects, for example, any outbound service calls made directly from the
+ * dialog implementation should be idempotent.
+ */
+public class DialogHost {
+
+    /**
+     * The... ummm... logger.
+     */
+    private static Logger logger = LoggerFactory.getLogger(DialogHost.class);
+    private static ObjectMapper objectMapper = new ObjectMapper()
```

Comment:
You missed this [comment](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/42.scaleout/DialogHost.cs#L24).